src/js/themes/Mender/light.js

```diff
@@ -78,6 +78,13 @@ export const light = {
           color: variantPalette.text.primary
         }
       }
+    },
+    MuiTooltip: {
+      styleOverrides: {
+        tooltip: {
+          backgroundColor: '#24444a'
```

Comment:
that could probably be defined in the `variantPalette` above to keep the color definitions in one place, OTOH it is a theme file already